semantic-analysis/src/Analysis/Concrete.hs

```diff
@@ -65,10 +65,6 @@ data Edge = Lexical | Import
   deriving (Eq, Ord, Show)
 
 
--- | Concrete evaluation of a term to a value.
---
---   >>> map fileBody (snd (concrete eval [File (Path.absRel "bool") (Span (Pos 1 1) (Pos 1 5)) (Core.bool True)]))
---   [Right (Bool True)]
```

Comment:
As described above, this was a very low-value test, and retaining it would have complicated things significantly, so 🔥